### ISSUE TITLE
Static pages links not working

### ISSUE THREAD

**Issue by sabino** [mangopik/Sximo-Laravel#58]
Hi.

I’m having a redirect problem. You can access my instance here: http://f-sabino.rhcloud.com

It is running under OpenShift, some adaptation was needed since I’ve used an already installed Laravel bundle provided by OpenShift. Database is not running on the same server.

It looks like is not permissions:
<img src="http://imgur.com/luBq8xKl.png">

**Comment by gamefanpage** [comment]
Did you enabled your mod_rewrite ? for allow htaccess

**Comment by sabino** [comment]
Yes, mod_rewrite is working. Just made a test with the following .htaccess content:

```
RewriteEngine On
Options +FollowSymLinks
RewriteRule ^test\.html http://www.google.com/? [R=301,L]
```

If you access http://f-sabino.rhcloud.com/test.html it is redirecting fine to Google.
